src/services/decoders/JsonlDecoder/index.ts

```
@@ -23,6 +24,9 @@
     convertToLogLevelValue,
     isJsonObject,
 } from "./utils";
+import {
+    parseFilterKeys,
+} from "../utils";
```

ESLint

[error] Imports must not be broken into multiple lines if there are 1 or less elements.

(import-newlines/enforce)